config/initializers/rpush.rb

```diff
@@ -0,0 +1,144 @@
+Rpush.configure do |config|
+  # Supported clients are :active_record and :redis
+  config.client = :redis
+
+  # Options passed to Redis.new
+  # config.redis_options = {}
+
+  # Frequency in seconds to check for new notifications.
+  config.push_poll = 2
```

Comment:
@fdoxyz How does this setting interact with https://github.com/forem/forem/pull/12419/files#r563913488? Since we're only pushing notifications every 30 seconds, is this potentially too high?

Reply:
It doesn't affect us because we're using the [RPush API](https://github.com/rpush/rpush/wiki/Push-API) (the recommended approach when run using a [scheduled job](https://github.com/rpush/rpush#in-a-scheduled-job)).  This config allows to configure the polling interval when rpush is a separate process, like sidekiq.

This is part of the defaults provided by the initial `bundle exec rpush init`. I don't think I edited the value provided either. This is similar to how Devise adds the initializer configs when installed. 

IMO this can be left as-is (or maybe commented out) since it shouldn't have any effect on the current implementation.